src/components/admin-complaints/index.js

```diff
@@ -668,6 +668,11 @@ class AdminComplains extends Component {
                                         selection 
                                         options={complaint_status_options} 
                                       />
+                                      {constants.statues.COMLAINT_STATUSES[complain.status]!='RESOLVED' && complain.remark && 
```

Comment:
Couldn't help but notice this typo, could you correct to "COMPLAINT"

Reply:
Working on it. The typo exists in multiple places, so I'm making a common PR for all these typo fixes.

Reply:
Added a issue for the typo, #27.